vendor/modules.txt

```diff
@@ -262,16 +262,19 @@ github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client/metadata
 github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/server
 github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/server/rpc/msg
 github.com/vmware/vsphere-automation-sdk-go/runtime/security
-# github.com/vmware/vsphere-automation-sdk-go/services/nsxt-vmc-aws-integration v0.2.0
+# github.com/vmware/vsphere-automation-sdk-go/services/nsxt-vmc-aws-integration v0.3.0
 github.com/vmware/vsphere-automation-sdk-go/services/nsxt-vmc-aws-integration/api
 github.com/vmware/vsphere-automation-sdk-go/services/nsxt-vmc-aws-integration/model
-# github.com/vmware/vsphere-automation-sdk-go/services/vmc v0.2.0
+# github.com/vmware/vsphere-automation-sdk-go/services/vmc v0.3.0
 github.com/vmware/vsphere-automation-sdk-go/services/vmc
 github.com/vmware/vsphere-automation-sdk-go/services/vmc/model
 github.com/vmware/vsphere-automation-sdk-go/services/vmc/orgs
 github.com/vmware/vsphere-automation-sdk-go/services/vmc/orgs/account_link
 github.com/vmware/vsphere-automation-sdk-go/services/vmc/orgs/sddcs
-# github.com/vmware/vsphere-automation-sdk-go/services/vmc/draas v0.2.0
+# github.com/vmware/vsphere-automation-sdk-go/services/vmc/autoscaler v0.1.0
+github.com/vmware/vsphere-automation-sdk-go/services/vmc/autoscaler/api/orgs/sddcs/clusters
+github.com/vmware/vsphere-automation-sdk-go/services/vmc/autoscaler/model
+# github.com/vmware/vsphere-automation-sdk-go/services/vmc/draas v0.3.0
```

Comment:
why do we have these commented imports?

Reply:
I have not manually modified this file. I only ran go mod vendor command